csrc/lower_thread_predicate.h

```diff
@@ -52,6 +52,8 @@ class TORCH_CUDA_CU_API ThreadPredicateMap {
     ParallelTypeBitmap limited_types;
     // Parallel types where only one thread/block is enough.
     ParallelTypeBitmap redundant_types;
+    // Map stores parallel types where a fraction of thread/block is enough
```

Comment:
Could you try to make this comment a bit more clear. I don't understand what the key-val pairs mean by this comment.